#
#
# patch "pidgin/plugins/gevolution/gevolution.c"
#  from [e82d07077f62e85a1e318e1f77fb093b2502e1a2]
#    to [f19cde741adeff1f88065dbd4c7580a49eed80ce]
#
============================================================
--- pidgin/plugins/gevolution/gevolution.c	e82d07077f62e85a1e318e1f77fb093b2502e1a2
+++ pidgin/plugins/gevolution/gevolution.c	f19cde741adeff1f88065dbd4c7580a49eed80ce
@@ -447,8 +447,8 @@ get_config_frame(PurplePlugin *plugin)
 	treeview = gtk_tree_view_new_with_model(GTK_TREE_MODEL(model));
 	gtk_tree_view_set_rules_hint(GTK_TREE_VIEW(treeview), TRUE);
 	gtk_box_pack_start(GTK_BOX(vbox),
-		pidgin_make_scrollable(treeview, GTK_POLICY_AUTO, GTK_POLICY_ALWAYS, GTK_SHADOW_IN, 300, 300),
-		TRUE, TRUE, 0);
+			pidgin_make_scrollable(treeview, GTK_POLICY_AUTOMATIC,
+			GTK_POLICY_ALWAYS, GTK_SHADOW_IN, 300, 300), TRUE, TRUE, 0);
 	gtk_widget_show(treeview);
 
 	/* Setup the column */
